database/insertData/03_template_A_featureShowcase.js

```diff
@@ -1102,6 +1102,161 @@ exports.queries = [
+                            "",
+                            [
+                              "user"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "users.nodes"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${firstName} \${lastName}"
+                          description: "\${username}"
+                        }
+                      }
+                    }
+                    {
+                      code: "searchCountries"
+                      index: 8
+                      title: "Search Demo (countries)"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Search for countries"
+                        description: "Type the two-character country code (must be CAPS sorry)"
+                        placeholder: "Search countries"
+                        icon: "world"
+                        minCharacters: 2
+                        multiSelect: true
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query countries($code: ID!) {country(code: $code) {name, capital, emoji, code}}",
+                            "https://countries.trevorblades.com",
+                            [
+                              "code"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "country"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${emoji} \${name}"
+                          description: "Capital: \${capital}"
+                        }
+                      }
+                    }
+                    {
+                      code: "searchATC"
+                      index: 9
+                      title: "Search ATC Codes"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Lookup ATC codes"
+                        description: "Start typing a drug name"
+                        placeholder: "Search medicines"
+                        icon: "pills"
+                        minCharacters: 3
+                        multiSelect: true
```

Comment:
While testing...

Something seemed wrong with the multi-selection or UC API calls... Not sure which. Can you investigate this please?

My test: typed "pam" and there were 4-5 results. And selected one:
![Screen Shot 2021-06-01 at 10 04 05 AM](https://user-images.githubusercontent.com/16461988/120245253-bfa98780-c2c0-11eb-81cd-13de26a2c384.png)

Then tried to search for another one start with "pam", it stays loading for ever... even if I erase what I type. It only goes back to normal if I refresh the page:
![Screen Shot 2021-06-01 at 10 03 48 AM](https://user-images.githubusercontent.com/16461988/120245250-bddfc400-c2c0-11eb-848d-120eddffffd1.png)

Reply:
Sure, this will be a front-end issue, I'll look into it for that PR.

Reply:
Done :) [Explanation](https://github.com/openmsupply/application-manager-web-app/pull/747#discussion_r642718661)

Thanks for catching this one :)